test/functional/managedclusterinfo_test.go

```diff
@@ -26,7 +26,9 @@ import (
 )
 
 const (
-	clusterDeploymentResponse = `# HELP acm_managed_cluster_info Managed cluster information
+	clusterDeploymentResponse = `# HELP acm_managed_cluster_addon_status_condition Managed cluster add-on status condition
+# TYPE acm_managed_cluster_addon_status_condition gauge
+# HELP acm_managed_cluster_info Managed cluster information
```

Comment:
Does this metrics need functional test?

Reply:
Functional test cases added for the new metrics.